src/main/java/com/hida/TokenType.java

```diff
@@ -0,0 +1,22 @@
+package com.hida;
+
+
+
+/**
+ * An Enum created to hold define the tokens used by the Minter class. Enums 

```

Comment:
Don't  need to go into the details of an `enum`.  A simpler description would be better, e.g. "The different types of tokens used by the Minter class."